### PR TITLE
Update to run fully through via FactStream

### PR DESCRIPTION
**Pull request by cguess**
This fully removes twitter (again) and then adds a few fixes to make sure Zenodotus can properly receive stuff from FactStream.

To test:
- `bundle install`
- Unfortunately it's really hard to test the ingestion from FactStream without a lot of set up so the best route I just `rails test` as usual.

### CODE REVIEW

**Review by reefdog — CHANGES_REQUESTED** [review]
@cguess There seems to be one accidentally change here that breaks tests (see inline comment). Also the [diff](https://github.com/TechAndCheck/zenodotus/pull/395/files) is a little weird… I wonder if some commits are missing / haven't been pushed yet? All I'm seeing is a change to the Sidekiq gem versioning pattern, removing some seed data, and removing the line that lets us scrape tweets in Archive.org URLs. The description implies there should be more going on?

File: app/media_sources/twitter_media_source.rb [hunk 1]
```diff
@@ -20,9 +20,7 @@ def self.valid_host_name
   # @returns [String or nil] the path of the screenshot if the screenshot was saved
   sig { override.params(url: String, force: T::Boolean).returns(T.any(T::Boolean, T::Hash[String, String])) }
   def self.extract(url, force = false)
-    url = MediaSource.extract_post_url_if_needed(url)
```
[the remark this answers is not in the document]
Comment (cguess) [reply app/media_sources/twitter_media_source.rb:23]: Ah yes, that was accidental, adding back (merging sucks!)

**Review by reefdog — APPROVED** [review]
All seems fine! Tests run, though as you noted I can't really test any further functionality than that. Still, 👍.

File: db/seeds.rb [hunk 2]
```diff
@@ -107,111 +107,3 @@
...
-  review_rating: {
-    "@type": "Rating",
-    "alternateName": "False",
-    "bestRating": "9",
-    "ratingValue": "4",
-    "worstRating": "0"
-  },
-  url: "https://www.realfact.com/factchecks/2021/feb/03/starwars",
-  media_review: media_review
-)
```
Comment (oneroyalace) [inline db/seeds.rb:217]: Why are we removing all of the seeds?

**Review by reefdog — CHANGES_REQUESTED** [review]
Actually yanking my approval on this one, I suspect the final state of it got borked during the merge or rebase from `master`. @cguess I think we should pair on the review of it when you have a moment!

**Comment by cguess** [comment]
> Actually yanking my approval on this one, I suspect the final state of it got borked during the merge or rebase from `master`. @cguess I think we should pair on the review of it when you have a moment!

Sounds good. Monday, first thing then

**Review by reefdog — APPROVED** [review]
It all runs clean now and I see new commits!

**Pull request merged**